build.gradle

```diff
@@ -4,15 +4,26 @@ buildscript {
     repositories {
         jcenter()
         mavenCentral()
+        maven { url 'https://maven.fabric.io/public' }
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'io.fabric.tools:gradle:1.+'
+        classpath 'com.google.gms:google-services:3.1.0'
```

Comment:
TODO: remove two lines

Reply:
@lorenzoPrimi you need to remove all that is related to fabric and I think also google-services. Otherwise f-droid will not compile the app.

Reply:
removed